
#######################################################################################################
                             PROJECT: tqdm
#######################################################################################################
CLONING PROJECT:
✓ SUCCESSFUL! This project is already cloned: https://github.com/tqdm/tqdm
.......................................................................................................
BUGGY LINES:
Suspicious lines to bug1:
41
Suspicious lines to bug2:
488,489,505,506,365
Suspicious lines to bug3:

Suspicious lines to bug4:
323
Suspicious lines to bug5:
769,770,771,772,773
Suspicious lines to bug6:
856
Suspicious lines to bug7:

Suspicious lines to bug8:
225
Suspicious lines to bug9:
The file is empty or does not exist: /Users/daniel/Bachelorarbeit/bugsinpy/projects/tqdm/bugs/9/bug_patch.txt
Not Found
.......................................................................................................

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 1

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: 8cc777fe8401a05d07f2c97e65d15e4460feab88
bug_file = /Users/daniel/Bachelorarbeit/Projects/tqdm/tqdm/contrib/__init__.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/tqdm/tqdm/tests/tests_contrib.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/tqdm/tqdm/contrib/__init__.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [41]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 41
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 3 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[]
✓ Number of created sketches:3
Time needed: 0.02 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 2

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: bef86db56654d271838b145ad77f7040a73a7b4d
bug_file = /Users/daniel/Bachelorarbeit/Projects/tqdm/tqdm/std.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/tqdm/tqdm/tests/tests_tqdm.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/tqdm/tqdm/std.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [365, 488, 489, 505, 506]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 365
x Fail! Line: 365 could not be transformed: 'and will possible be set to postfix = ', ' + postfix.'
Start transforming line: 488
✓
Start transforming line: 489
✓
Start transforming line: 505
✓
Start transforming line: 506
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 191 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[365]
✓ Number of created sketches:191
Time needed: 0.05 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 3

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
x FAIL! Suspicious lines are empty!

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 4

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: 03b347646492131d889871939b40457d29147216
bug_file = /Users/daniel/Bachelorarbeit/Projects/tqdm/tqdm/_tqdm.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/tqdm/tqdm/tests/tests_tqdm.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/tqdm/tqdm/_tqdm.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [323]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 323
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 4 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[]
✓ Number of created sketches:4
Time needed: 0.01 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 5

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: 19b08ab34fdbfa0275bc5cb2430436c724c7e759
bug_file = /Users/daniel/Bachelorarbeit/Projects/tqdm/tqdm/_tqdm.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/tqdm/tqdm/tests/tests_tqdm.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/tqdm/tqdm/_tqdm.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [769, 770, 771, 772, 773]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 769
✓
Start transforming line: 770
x Fail! Line: 770 could not be transformed: 'try'
Start transforming line: 771
✓
Start transforming line: 772
x Fail! Line: 772 could not be transformed: 'except (TypeError, AttributeError)'
Start transforming line: 773
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 107 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[770, 772]
✓ Number of created sketches:107
Time needed: 0.04 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 6

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: a4b9c86db548b2d0dbb5af7a6bbdc26ab47e1eec
bug_file = /Users/daniel/Bachelorarbeit/Projects/tqdm/tqdm/_tqdm.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/tqdm/tqdm/tests/tests_synchronisation.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/tqdm/tqdm/_tqdm.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [856]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 856
x Fail! Line: 856 could not be transformed: ' self.total)'
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 0 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[856]
✓ Number of created sketches:0
Time needed: 0.01 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 7

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
x FAIL! Suspicious lines are empty!

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 8

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
GIT RESET TO: 08b8ad1ff3bd003ef8309faaa0cc108ffa40317d
bug_file = /Users/daniel/Bachelorarbeit/Projects/tqdm/tqdm/_tqdm.py
bug_test_file = /Users/daniel/Bachelorarbeit/Projects/tqdm/tqdm/tests/tests_tqdm.py
TEST EXECUTION:
✓ SUCCESSFUL! TEST PASSES
✓ PySketchFix is started...
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

	                              PySketchFix®: Sketches are created.

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
PySketchFix will run with:
bug file: /Users/daniel/Bachelorarbeit/Projects/tqdm/tqdm/_tqdm.py
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The suspicious lines which are taken are: [225]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start transforming line: 225
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Start creating 1 sketches.
✓
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
x Lines where no transformation is found:[]
✓ Number of created sketches:1
Time needed: 0.04 minutes.

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+

                                          BUG: 9

+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+*+
x FAIL! Suspicious lines are empty!
#######################################################################################################
Results with bug numbers:
Project:tqdm
Failing:[]
Passing:[1, 2, 4, 5, 6, 8]
Empty Suspicious Lines:[3, 7, 9]
No bug information is found:[]
